InteractionController.cs

```diff
@@ -70,6 +86,66 @@ void PrintBooks(List<Book> books)
             .ForEach(Console.WriteLine);
     }
 
+    // from BookHelper
+    static void PrintUpdatedBookProperties(string updatedBook)
+    {
+        Console.WriteLine("\nUpdated. New properties are as follows: ");
+        Console.WriteLine(ConvertLineToPropertiesList(updatedBook));
+    }
+
+    static public string ConvertLineToPropertiesList(string line)
+    {
+        string[] properties = line.Split(',');
+        return $"1. Title: {properties[0]}\n2. Author: {properties[1]}\n3. Year published: {int.Parse(properties[2])}";
```

Comment:
Array accessor, don't use

Reply:
Have replaced with `line.Split(','); ` and list accessor